tesseract_command_language/include/tesseract_command_language/joint_waypoint.h

```diff
@@ -287,10 +287,9 @@ class JointWaypoint
     if (lower_tolerance.size() == 0 || upper_tolerance.size() == 0)
       return false;
 
-    // Check if they are close to 0
-    Eigen::VectorXd range = upper_tolerance - lower_tolerance;
-    double sum = range.sum();
-    return (sum > 1e-5);
+    // Check if they are the same
+    static auto max_diff = static_cast<double>(std::numeric_limits<float>::epsilon());
+    return !tesseract_common::almostEqualRelativeAndAbs(lower_tolerance, upper_tolerance, max_diff);
```

Comment:
```suggestion
    // Check if the upper tolerance is greater than the lower tolerance by some small amount
    return ((upper_tolerance - lower_tolerance).array() > max_diff).all();
```

Reply:
@marip8 The changes discussed in our meeting have been applied. Let me know if you see an issues with the changes.